torch/csrc/jit/runtime/decomposition_registry.cpp

```diff
@@ -8,7 +8,9 @@
 #include <torch/csrc/jit/serialization/import_source.h>
 
 #include <c10/util/Exception.h>
+#include <jit/runtime/graph_executor.h>
```

Comment:
An error from this line broke internal builds on the associated diff (D35971232). We attempted a forward-fix but it could not be cleanly patched due to further changes to this file. We will need to revert, and we can re-land this with just the one-line change below.

```suggestion
#include <torch/csrc/jit/runtime/graph_executor.h>
```

Reply:
(I'm happy to stamp the resubmitted PR once this change is made)